src/water_cmd.cpp

```diff
@@ -427,6 +427,15 @@ bool RiverModifyDesertZone(TileIndex tile, void *)
 	return false;
 }
 
+void DoRiverModifyDesertZone(TileIndex tile, uint8 random_bits) {
+	MakeRiver(tile, random_bits);
```

Comment:
Instead of passing Random() as a parameter into random_bits, why not just call Random() here? (Honest question, I haven't done much with randomness and may be missing something.)

Reply:
I agree, remove the `random_bits` param and just use `Random()` in MakeRiver call.